source/globalCommands.py

```diff
@@ -567,6 +566,26 @@ def script_toggleReportTableCellCoords(self,gesture):
 	script_toggleReportTableCellCoords.__doc__=_("Toggles on and off the reporting of table cell coordinates")
 	script_toggleReportTableCellCoords.category=SCRCAT_DOCUMENTFORMATTING
 
+	@script(
+		# Translators: Input help mode message for toggle report cell borders command.
+		description=_("Cycles through cell borders settings"),
+		category=SCRCAT_DOCUMENTFORMATTING,
+	)
+	def script_toggleReportCellBorders(self, gesture):
+		if not config.conf["documentFormatting"]["reportBorderStyle"] and not config.conf["documentFormatting"]["reportBorderColor"]:
+			# Translators: A message reported when cycling through cell borders settings.
+			config.conf["documentFormatting"]["reportBorderStyle"] = True

```

Comment:
At the moment check for translatable commends fails because the commend is not placed directly above the message.